### PR TITLE
Fix translation links

### PR DESCRIPTION
**Pull request by shushugah**
This removes about 150 auto-generated "translated files" (from 254 down to 99), which are basically English files. For example https://techworkerscoalition.org/es/summer-learning-club/ and https://techworkerscoalition.org/it/summer-learning-club/ are valid links, but are using exact same content as English file.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ gem 'pry'
 # If you have any plugins, put them here!
 group :jekyll_plugins do
    gem "jekyll-feed", "~> 0.11"
-   gem "jekyll-multiple-languages-plugin"
+   gem "jekyll-multiple-languages-plugin", github: "kurtsson/jekyll-multiple-languages-plugin", ref: "refs/pull/178/head"
```
Comment (shushugah) [inline Gemfile:23]: This is necessary, to exclude the English language version of `/brasil` which does not exist, as only `/pt/brasil` exists. When the gem version is updated, we can point to that.

File: _config.yml [hunk 2]
```diff
@@ -31,7 +31,15 @@ jekyll_get_json:
...
 exclude:
   - .jekyll-cache/
+# also update index.md
 languages: ["en", "es", "fr", "it", "pt", "ru"]
+defaults:
+  - scope:
+      path: ""
+    values:
+      # unless languages: ["bl" "ah"] is included, english is only/default frontmatter language
+      languages: ["en"]
```
Comment (shushugah) [inline _config.yml:41]: applies a default of English for all pages. And it's overrode per file, with custom values

File: _data/chapters.yaml [hunk 3]
```diff
@@ -7,7 +7,7 @@ regions:
       meetup: Tech-Workers-Coalition-ATX/
       facebook: twcatx
     - text: Brasil
-      url: /brasil
+      url: /pt/brasil
```
Comment (shushugah) [inline _data/chapters.yaml:10]: /brasil doesn't/shouldn't exist, unless it's in English.

File: _i18n/pt.yml [hunk 4]
```diff
@@ -21,7 +21,7 @@ home:
   how_we_work:
     title: Como trabalhamos
     description: |
-      Somos uma organização democraticamente estruturada, totalmente voluntária e liderada por trabalhadores. Nesse momento, a associação consiste em participar de reuniões pessoalmente e trabalhar em vários projetos nos quais as pessoas estão interessadas. Organizamos on-line, mas os encontros em vida real são o ponto crucial do que fazemos. Confira nosso <a href="/pt/guia-da-comunidade">Guia da comunidade</a>  para mais informações. Também entre no <a href="/subscribe">Slack</a> para se conectar com pessoas em sua região.
+      Somos uma organização democraticamente estruturada, totalmente voluntária e liderada por trabalhadores. Nesse momento, a associação consiste em participar de reuniões pessoalmente e trabalhar em vários projetos nos quais as pessoas estão interessadas. Organizamos on-line, mas os encontros em vida real são o ponto crucial do que fazemos. Confira nosso <a href="/pt/guia-da-comunidade/">Guia da comunidade</a>  para mais informações. Também entre no <a href="/subscribe">Slack</a> para se conectar com pessoas em sua região.
```
Comment (shushugah) [inline _i18n/pt.yml:24]: I cannot figure out why, but `techworkerscoalition.org/pt/guia-da-comunidade/` does not work...

**Review by rossPatton — APPROVED** [review]

**Pull request merged**